### DISCUSSION TITLE
Feature Request: SMTP Account Grouping and Hourly/Daily Rate Restrictions

### DISCUSSION THREAD

**Discussion by sockpapi** [parcelvoy/platform#285] [Ideas]
In today's dynamic email marketing landscape, one widespread strategy involves the utilization of numerous Google Workspace email accounts, each typically sending around 50 emails daily.

Given the limited number of emails sent from each account, there is a necessity to consolidate these accounts into groups. This would enable a more substantial daily email output. Hence, the request for the ability to group these SMTP accounts and implement hourly/daily rate limits is of utmost importance.

**Comment by pushchris** [comment]
Appreciate the feature request, we'll add it to the backlog! We are currently focusing on functionality to bring internal marketing up to the level of the main players in the space. This functionally seems like it's focused on external marketing or campaigns to folks who aren't a companies user base. I can definitely see us expanding into that area in the future, but it'll probably be a bit.

**Comment by pushchris** [comment]
@sockpapi we've added a load balancer step to our journey builder that may actually be able to be used for what you are looking for here. You could create a journey that splits into three for example each sending an email from a different sender and rate limit to your example of fifty a day.

**Comment by leobarcellos** [comment]
#485

This PR add the rate interval feature, which make the hourly/daily possible on rate limit
